File: app/src/main/java/com/ontherunvaro/onoclient/MONOApp.java
package com.ontherunvaro.onoclient;

import android.app.Application;
import android.util.Log;

import com.ontherunvaro.onoclient.util.ConfigUtil;

/**
 * Created by varo on 19/11/16.
 */

public class MONOApp extends Application {

    private static final String TAG = "MONOApp";

    @Override
    public void onCreate() {
        super.onCreate();
        Log.d(TAG, "onCreate: MONO started");
        //initialize config
        ConfigUtil.load(this);
    }
}

File: app/src/main/java/com/ontherunvaro/onoclient/MONOApp.kt
/*
 * mONO is a free app for a telephony provider's client area.
 * Copyright (C) 2017 Álvaro Brey Vilas <alvaro.brv@gmail.com>
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <https://www.gnu.org/licenses/gpl-3.0>.
 */

package com.ontherunvaro.onoclient

import android.app.Application
import android.util.Log

class MONOApp : Application() {

    override fun onCreate() {
        super.onCreate()
        Log.d(TAG, "onCreate: mONO started")
    }

    companion object {
        private val TAG = "MONOApp"
    }
}
